File: ls/server/src/main/java/org/opencds/cqf/cql/ls/server/command/CliCommand.java
package org.opencds.cqf.cql.ls.server.command;

import org.opencds.cqf.cql.ls.server.manager.IgContextManager;
import picocli.CommandLine.Command;

@Command(subcommands = {CqlCommand.class})
public class CliCommand {
    private IgContextManager igContextManager;

    public IgContextManager getIgContextManager() {
        return this.igContextManager;
    }

    public CliCommand(IgContextManager igContextManager) {
        this.igContextManager = igContextManager;
    }
}

File: ls/server/src/main/kotlin/org/opencds/cqf/cql/ls/server/command/CliCommand.kt
package org.opencds.cqf.cql.ls.server.command

import org.opencds.cqf.cql.ls.server.manager.IgContextManager
import picocli.CommandLine.Command

@Command(subcommands = [CqlCommand::class])
class CliCommand(val igContextManager: IgContextManager)
